index.js

```diff
@@ -23,10 +23,16 @@ try {
   // Fetch inputs
   const name = core.getInput('name');
   const environment = core.getInput('environment');
+  const domain = core.getInput('domain');
+  const output = branchName(github.context).toLowerCase().replace(/[^a-z0-9-]/g, "").substr(0, 63)
```

Comment:
You can always DRY `toLowerCase....` to `trim` function or similar... Works like this though

Reply:
Ash missed this last time - rename this variable to `branch`, makes more sense than just generic `output` - especially as there is multiple outputs here... :)

Reply:
Ash?

Reply:
Haha, auto-correct, read as `ahh` :D